src/pages/sign-up/Email/index.tsx

```diff
@@ -104,6 +105,20 @@ export const Email = () => {
               <Google />
               Continue with Google
             </Button>
+            <Button
+              color="primary"
+              variant="outlined"
+              onClick={() => {
+                tried();
+                type === 'user'
+                  ? navigate(`/oauth/apple${eventName && `?event_name=${eventName}`}`)
+                  : navigate('/oauth/apple');
+              }}
+              style={{ display: 'flex', gap: '12px', width: '100%', marginTop: '12px' }}
```

Comment:
why inline-style? please do not use inline style. 
please use `pre-defined tailwind classes or `module.scss`